libs/utils/src/lib/format/number.ts

```diff
@@ -90,6 +90,17 @@ export const formatNumberFixed = (
   return getFixedNumberFormat(formatDecimals).format(Number(rawValue));
 };
 
+export const addDecimalsFormatNumberQuantum = (
+  rawValue: string | number,
+  decimalPlaces: number,
+  quantum: number | string
+) =>
+  addDecimalsFormatNumber(
+    rawValue,
+    decimalPlaces,
+    Math.max(0, Math.log10(100 / Number(quantum) || 0))
```

Comment:
**chore (non-blocking)**: Sounding like a broken record, but I'd also test this function for this line only. It takes some effort to pull apart what it's doing.

Reply:
added more tests